src/firefly/js/util/Color.js

```diff
@@ -14,7 +14,7 @@ export const toRGB= (color) => chroma.valid(color) ? chroma(color).rgb() : [0,0,
 export const getRGBA= (color)  => chroma.valid(color) ? chroma(color).rgba() : [0,0,0,1];
 
 export const hexColorWithAlpha = (color, alpha) => chroma.valid(color) ? chroma(color).alpha(alpha).hex() : color;
-
+export const joyVarColorWithAlpha = (joyVarColor, alpha) => hexColorWithAlpha(joyVarColor.split(', ')?.[1]?.slice(0, -1) ?? joyVarColor, alpha);
```

Comment:
good idea! There are a couple of other joy related things things that we could put here as well.